lib/settings_page.dart

```diff
@@ -0,0 +1,278 @@
+              new Container(
+                height: 90,
+                child: Padding(
+                    padding: EdgeInsets.only(
+                      top: 30,
+                    ),
+                    child: offTopTitle1),
+              ),
+              new Container(
+                color: Colors.white24,
+                height: 100,
+                width: 100,
+                child: CircleAvatar(
+                    backgroundImage:
+                        AssetImage('assets/images/Off-Top_Logo.jpg')),
+              ),
+              new Container(
+                  height: (100),
+                  width: MediaQuery.of(context).size.width,
+                  color: Colors.white,
+                  child: new ButtonBar(
+                    alignment: MainAxisAlignment.spaceBetween,
+                    children: <Widget>[
+                      new FlatButton(
+                        onPressed: () => show1(),
+                        padding: EdgeInsets.fromLTRB(20, 0, 0, 0),
+                        child: Text("Profile",
+                            style: TextStyle(
+                                fontSize: 30.0,
+                                color: Colors.black,
+                                fontWeight: FontWeight.w400)),
+                      ),
+                      new FlatButton(
+                        onPressed: () => show2(),
+                        padding: EdgeInsets.fromLTRB(60, 0, 20, 0),
+                        child: Text("Settings",
+                            style: TextStyle(
+                                fontSize: 30.0,
+                                color: Colors.black,
+                                fontWeight: FontWeight.w400)),
+                      )
+                    ],
+                  )),
+              Visibility(
+                child: new Container(
+                    color: Colors.transparent,
+                    height: (MediaQuery.of(context).size.height -
+                        customBar.preferredSize.height -
+                        320),
+                    width: (MediaQuery.of(context).size.width - 50),
+                    child: _buildProfileForm()),
+                visible: _showContainer,
+              ),
+              Visibility(
+                child: new Container(
+                  height: (MediaQuery.of(context).size.height -
+                      customBar.preferredSize.height -
+                      320),
+                  child: _buildNestedListView(),
+                ),
+                visible: _showSettings,
+              )
+            ]))));
+  }
+
+  Widget _buildNestedListView() {
+    return ListView(children: <Widget>[
+      new ExpansionTile(
+        leading: Icon(Icons.line_style),
+        title: Text('DEFAULT CATEGORY',
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 15)),
+        
+        children: <Widget>[
+          new ListTile(
```

Comment:
this ListTile() and the one on line 156, 183, 211 have the same code just different text. You can make a modular method where you pass you're desired text into these fields. This article has an idea of what you want to do, although not completely as there's some code and features in this article that we don't want to add. However the basic idea is that you want to be creating a reusable widget for these ListTiles() [https://itnext.io/building-your-first-reusable-widget-with-flutter-cadb54c3c253](url)